src/main/java/emissary/place/ServiceProviderPlace.java

```diff
@@ -218,6 +222,15 @@ protected ServiceProviderPlace(InputStream configStream, String placeLocation) t
         this(configStream, null, placeLocation);
     }
 
+    /**
+     * Create the place based on another ServiceProviderPlace
+     *
+     * @param place the ServiceProviderRefreshablePlace to clone
+     */
+    protected ServiceProviderPlace(final ServiceProviderPlace place) {
```

Comment:
Should this take a ServiceProviderRefreshablePlace as an argument?

Reply:
ServiceProviderPlace doesn't know anything about ServiceProviderRefreshablePlace. I was having issues with the `ServiceProviderRefreshablePlace(ServiceProviderRefreshablePlace place)` constructor and java compiler adding `super()` which was calling `ServiceProviderPlace()` and causing unintended side-effects. I had to add this little ditty to get past it. I ❤️  Java.